### ISSUE TITLE
What is the effect of frozen layers on GMACs?

### ISSUE THREAD

**Issue by ahmadmobeen** [sovrasov/flops-counter.pytorch#75]
The GMACs are the same regardless of any frozen layers. 
```
All layers are trainable:
Computational complexity:       7.63 GMac
Number of parameters:           128.92 M

Only classifier is trainable:
Computational complexity:       7.63 GMac
Number of parameters:           155.69 k
```

In my understanding, if "param.requires_grad' is set to 'False' in some of the layers, those layers would not be computed however they would remain part of the graph. 

So, in the calculation of GMACs, such layers should be excluded as they would not be computed during the training hence reducing the number of operations? 

Please correct me if my understanding is wrong.

**Comment by sovrasov** [comment]
`requires_grad` prevents pytorch from computing gradients for particular parameters during training. This flag doesn't affect forward pass complexity, which is measured by ptflops. See pytorch docs for datails.
